internal/model/probe.go

```diff
@@ -0,0 +1,23 @@
+package model
+
+// Probe is the model required by the service to manipulate
+// the resource.
+type Probe struct {
+	Name   string
+	URL    string
+	Status string
+	Delay  uint
+	Finish chan bool
+	ID     int
+}
+
+func (p Probe) Copy() Probe {
+	return Probe{
+		Name:   p.Name,
+		URL:    p.URL,
+		Status: p.Status,
+		Delay:  p.Delay,
+		Finish: p.Finish,
+		ID:     p.ID,
+	}
+}
```

Comment:
Why do you need a copy ?

Reply:
I needed one because I was dealing with pointers on the initWithStoredProbes function, I'll try to find a more obvious solution